docs/complementary-tools.md

```diff
@@ -23,6 +23,7 @@
 - [GraphQL PHP Validation Toolkit](https://github.com/shmax/graphql-php-validation-toolkit) - Small library for validation of user input
 - [MLL Scalars](https://github.com/mll-lab/graphql-php-scalars) - Collection of custom scalar types
 - [X GraphQL](https://github.com/x-graphql) - Provides set of libraries for building http schema, schema gateway, transforming schema, generating PHP code from execution definition, and more.
+- [GraphQL Attribute Schema](https://github.com/jerowork/graphql-attribute-schema) - Easily build your GraphQL schema for webonyx/graphql-php using PHP attributes
```

Comment:
```suggestion
- [GraphQL Attribute Schema](https://github.com/jerowork/graphql-attribute-schema) - Build your GraphQL schema for webonyx/graphql-php using PHP attributes instead of array-based configuration
```

See https://github.com/jerowork/graphql-attribute-schema/pull/7